### PR TITLE
Mis Pato Cambios

### PR DESCRIPTION
**Pull request by patolinbr**
Wait call view

### CODE REVIEW

**Review by patolinbr — COMMENTED** [review]
Good from Pato

File: package.json [hunk 1]
```diff
@@ -23,16 +23,19 @@
     "@magidoc/cli": "^4.1.4",
     "@nextui-org/react": "^2.2.10",
     "aws-amplify": "^6.0.20",
+    "bootstrap": "^5.3.3",
```
Comment (ronuma) [inline package.json:26]: IS THIS NECESSARY?

File: src/app/(main)/@agent/wait/page.tsx [hunk 2]
```diff
@@ -1,9 +1,27 @@
 import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import Button from 'react-bootstrap/Button'
+import Spinner from 'react-bootstrap/Spinner'
+import Image from 'next/image'
 
-function page() {
+function wait() {
```
Comment (ronuma) [inline src/app/(main)/@agent/wait/page.tsx:7]: CamelCase for components

File: src/app/(main)/@agent/wait/page.tsx [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
   )
 }
 
-export default page
\ No newline at end of file
+export default wait
```
Comment (ronuma) [inline src/app/(main)/@agent/wait/page.tsx:26]: CamelCase

**Review by ronuma — CHANGES_REQUESTED** [review]
Also, pr name and description we have no clue

File: src/app/(main)/@agent/wait/page.tsx (hunk 2 above)
Comment (IksGarG) [reply src/app/(main)/@agent/wait/page.tsx:7]: Fixed

File: src/app/(main)/@agent/wait/page.tsx (hunk 3 above)
Comment (IksGarG) [reply src/app/(main)/@agent/wait/page.tsx:26]: Fixed

**Review by IksGarG — APPROVED** [review]
Fixed changes, but need to work with @patolinbr to NOT INSTALL LIBRARIES IF NOT REQUIERD.

**Pull request merged**